Directory.Packages.props

```diff
@@ -90,4 +95,7 @@
     <PackageVersion Include="Microsoft.VariantAssignment.Client" Version="2.4.17140001" />
     <PackageVersion Include="Microsoft.VariantAssignment.Contract" Version="3.0.16990001" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageVersion Update="CommunityToolkit.WinUI.Collections" Version="8.0.240109" />
```

Comment:
Why are we adding this package? And out of the list? Looks like it wasn't intended.